appwrite/website
# Add blog post: Deno 2.0 and what it means for Appwrite Functions

## What does this PR do?

- Adds Deno 2.0 blog post
- Add blog post assets

## Test Plan

- Run the website and navigate to `/blog/post/deno-2-appwrite-functions`
- Confirm that blog post exists and that the assets are displayed correctly

## Related PRs and Issues

None

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

Yes
